targets/x86-xen_domu/profiles.mk

```diff
@@ -0,0 +1,6 @@
+X86_XEN_MODULES := 
+
+$(eval $(call GluonProfile,GENERIC,$(X86_XEN_MODULES)))
```

Comment:
Please just remove the second argument to GluonProfile instead of defining an unused variable.